input/resources/CodeSystem-name-context.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
```

Comment:
Should have a registered OID as an identifier, see https://github.com/hl7au/au-fhir-base/wiki#terminology-resource-conventions